service/static/js/index.js

```diff
@@ -324,4 +324,100 @@ $(function () {
     });
   });
 
+  // ****************************************
+  // Delete a Product
+  // ****************************************
+
+  $("#delete-product").click(function (event) {
+    event.preventDefault();
+    var product_id = $("#wishlist-product-id").val();
+    var wishlist_id = $("#wishlist-id").val();
+
+    var ajax = $.ajax({
+      type: "DELETE",
+      url: `/wishlists/` + wishlist_id + `/products/` + product_id,
+      contentType: "application/json",
+      data: '',
+    })
+
+    ajax.done(function(res){
+      clear_form_data();
+      flash_message("Success");
+    });
+
+    ajax.fail(function(res){
+      flash_message("Product doesn't exist!");
+      clear_table();
+    });
+  });
+
+  // ****************************************
+  // Update a Product
+  // ****************************************
+  $("#update-product").click(function (event) {
+    event.preventDefault();
+    var wishlist_id = $("#wishlist-id").val();
+    var name = $("#wishlist-product-name").val();
+    var product_id = $("#wishlist-product-id").val();
+    var i_id = $("#wishlist-inventory-product-id").val();
+    var price = $("#wishlist-product-price").val();
+    var status = $("#wishlist-product-status").val();
+    var pic = $("#wishlist-product-pic").val();
+    var desc = $("#wishlist-product-description").val();
+
+    // var data = {
+    //   ...name && {name = name},
+    //   ...i_id && {inventory_product_id = parseInt(i_id)},
+    //   ...price && {price = parseFloat(price)},
+    //   ...(status == "AVAILABLE") && {status = 1},
+    //   ...(status == "UNAVAILABLE") && {status = 0},
+    //   ...pic && {pic_url = pic},
+    //   ...desc && {short_desc = desc}
+    // };
```

Comment:
Did this not work? 😓

Reply:
It will affects other part. I don't know why.

Reply:
Let me try